src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/loadbalancer/ApacheEc2LoadBalancerFormula.kt

```diff
@@ -56,6 +60,14 @@ internal class ApacheEc2LoadBalancerFormula : LoadBalancerFormula {
         )
     }
 
+    override fun updateJiraConfiguration(ssh: Ssh, unpackedProduct: String) {
+        ssh.newConnection().use { shell ->
+            Sed().replace(shell, "bindOnInit=\"false\"",
```

Comment:
IMO we use slightly different code style in the project:

```
Sed().replace(
    shell,
    "bindOnInit=\"false\"",
    "bindOnInit=\"false\" scheme=\"http\" proxyName=\"${uri?.host}\" proxyPort=\"80\"",
    "$unpackedProduct/conf/server.xml"
)
```

Reply:
Done.